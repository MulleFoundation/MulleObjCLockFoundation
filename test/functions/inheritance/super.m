#import <MulleObjC/MulleObjC.h>
#import <MulleObjC/NSDebug.h>

#include "class-system.inc"


int   main( int argc, char *argv[])
{
   MulleObjCDotdumpMetaHierarchy( "B");

   [A super];
   printf( "\n");
   [B super];
   return( 0);
}

