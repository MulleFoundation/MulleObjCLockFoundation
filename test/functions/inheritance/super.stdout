+[A super]

+[B(Y) super]
+[Q super]
+[P super]
+[A super]
